docs/dev/connectors/kinesis.md

```diff
@@ -271,9 +271,9 @@ For the monitoring to work, the user accessing the stream needs access to the Cl
 {% highlight java %}
 Properties producerConfig = new Properties();
 // Required configs
-producerConfig.put(AWSConfigConstants.AWS_REGION, "us-east-1");
 producerConfig.put(AWSConfigConstants.AWS_ACCESS_KEY_ID, "aws_access_key_id");
 producerConfig.put(AWSConfigConstants.AWS_SECRET_ACCESS_KEY, "aws_secret_access_key");
+producerConfig.put("Region", "us-east-1");
```

Comment:
Is the "Region" string defined in some KPL class, e.g. `XXConstants`?
If yes, maybe we should just demonstrate that class.
That would make it more clear that the KPL configuration keys are directly supported.

Reply:
no, KPL doesn't have such configs. KPL takes a string like 'Region' and tries to find its setter using reflection.